lnbits/core/views/public_api.py

```diff
@@ -16,10 +17,17 @@
 
 @core_app.get("/.well-known/lnurlp/{username}")
 async def lnaddress(username: str, request: Request):
-    from lnbits.extensions.lnaddress.lnurl import lnurl_response  # type: ignore
+    if "lnaddress" not in settings.lnbits_disabled_extensions:
+        from lnbits.extensions.lnaddress.lnurl import lnurl_response # type: ignore
 
-    domain = urlparse(str(request.url)).netloc
-    return await lnurl_response(username, domain, request)
+        domain = urlparse(str(request.url)).netloc
+        return await lnurl_response(username, domain, request)
+
+    elif "lnaddy" not in settings.lnbits_disabled_extensions:
```

Comment:
This means that it is either handled by `lndaddress` **or** `lnaddy`, is this intended?

Reply:
There are two extensions that handle lightning addresses, there the one in this pull request (the free admin lnaddress) and then the cloudflare paid lightning extension. 

I was not sure how you want to address this conflict, as I originally wrote this as a personal extension, but right now as it stands to avoid name conflict on the local instance, the lnaddress extension has to be disabled in order for the lnaddy extension to work. 

If you feel there is a better way around this like checking both and allowing both extensions to be active, then we should do it.

Reply:
I added a note in the .example.env and will add a note to the README.md in extension repo over here: https://github.com/lnbits/free-lnaddress-extension